### ISSUE TITLE
Not enough contrast for some text

### ISSUE THREAD

**Issue by tagawa** [w3c/Mobile-Checker#78]
To pass at least WCAG AA, some text seems to need more contrast, e.g. the resolution figures in the device-selector buttons on the front page. There may be more so it would be worth running a contrast checker on all pages.

**Comment by guibbs** [comment]
Hi, thanks for feedback @tagawa . I am looking for a contrast checker now. Do you have a good one in mind?

**Comment by tagawa** [comment]
I usually use the [Juicy Studio Accessibility Toolbar](https://addons.mozilla.org/en-US/firefox/addon/juicy-studio-accessibility-too/) for Firefox which checks a whole page, although I concentrate on luminosity for WCAG 2. Then for individual colours I'll use [this contrast checker](http://www.snook.ca/technical/colour_contrast/colour.html).

I've just sent a pull request (https://github.com/w3c/Mobile-Checker/pull/79) which has improved the luminosity to pass WCAG 2. The change is slight so it shouldn't disrupt the current colour scheme.

**Comment by guibbs** [comment]
Thank you @tagawa! I'll keep these tools in mind.

**Comment by guibbs** [comment]
I'll run it on the report page too.

**Comment by tagawa** [comment]
Good thinking - I forgot about that.
And thanks for the merge.

**Comment by edent** [comment]
Adding to this.  The text on the front page is extremely low contrast.
![contrast](https://cloud.githubusercontent.com/assets/837136/8542086/d8b2bb78-248a-11e5-9f8e-48b26d77bc04.png)

The link on the left is being hovered - it has a ratio of 1.4

- https://leaverou.github.io/contrast-ratio/#%232a6496-on-%23334d5c

The link on the right is in the normal state - the ratio is a marginally better 2.0

- https://leaverou.github.io/contrast-ratio/#%233f7db1-on-%23334d5c

As per best practice, the ratio should be 3:1 or better - http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G183

**Comment by dontcallmedom** [comment]
Hi @edent we've released a new version that has much better contrast now. Thanks!

I'm closing this issue, but if you find other text that needs better contrast, please do open another one.
